syntax = "proto3";
import "common.proto";
package gw2l;
option cc_generic_services = true;

message LoginRequest
{
  string account = 1;
  string password = 2;
}

message LoginResponse
{
	ClientError error = 1;
}

service LoginService {
  rpc Solve (LoginRequest) returns (LoginResponse);
}
